samples/iolib/src/main/cpp/player/SampleSource.h

```diff
@@ -57,10 +57,20 @@ class SampleSource: public DataSource {
         }
     }
 
+    void play() {
+        mIsPlaying = true;
```

Comment:
I'd suggest to call setPlayMode(false) here instead of modifying the `mIsPlaying`. Same for below.

I'm okay with the change. But would like to have Robert to comment here as he is the owner for code here.

Reply:
I agree that we should call setPlayMode. Otherwise, we lose the fromPause parameter